core/src/test/scala/org/http4s/rho/ParamDefaultValueSpec.scala

```diff
@@ -6,340 +6,340 @@ import org.specs2.mutable.Specification
 
 class ParamDefaultValueSpec extends Specification {
 
-  def body(service: HttpRoutes[IO], r: Request[IO]): String =
-    new String(service(r).value.unsafeRunSync().getOrElse(Response.notFound).body.compile.toVector.unsafeRunSync().foldLeft(Array[Byte]())(_ :+ _))
+  def body(routes: HttpRoutes[IO], r: Request[IO]): String =
+    new String(routes(r).value.unsafeRunSync().getOrElse(Response.notFound).body.compile.toVector.unsafeRunSync().foldLeft(Array[Byte]())(_ :+ _))
 
   def requestGet(s: String, h: Header*): Request[IO] =
     Request(bits.MethodAliases.GET, Uri.fromString(s).right.getOrElse(sys.error("Failed.")), headers = Headers(h: _*))
 
   "GET /test1" should {
-    val service = new RhoService[IO] {
+    val routes = new RhoService[IO] {
       GET / "test1" +? param[String]("param1") |>> { param1: String => Ok("test1:" + param1) }
     }.toRoutes()
 
     "map parameter with default value" in {
-      body(service, requestGet("/test1")) must be equalTo "Missing query param: param1"
+      body(routes, requestGet("/test1")) must be equalTo "Missing query param: param1"
     }
     "map parameter with empty value" in {
-      body(service, requestGet("/test1?param1=")) must be equalTo "test1:"
+      body(routes, requestGet("/test1?param1=")) must be equalTo "test1:"
     }
     "map parameter with value" in {
-      body(service, requestGet("/test1?param1=value1")) must be equalTo "test1:value1"
+      body(routes, requestGet("/test1?param1=value1")) must be equalTo "test1:value1"
     }
     "map parameter without value" in {
-      body(service, requestGet("/test1?param1")) must be equalTo "Value of query parameter 'param1' missing"
+      body(routes, requestGet("/test1?param1")) must be equalTo "Value of query parameter 'param1' missing"
     }
   }
 
   "GET /test2" should {
-    val service = new RhoService[IO] {
+    val routes = new RhoService[IO] {
```

Comment:
This is still a `RhoService`, so i don't know that the rename is appropriate here.